parser.go

```diff
@@ -1,10 +1,14 @@
 package enmime
 
+// ReadPartErrorPolicy allows to recover the buffer (or not) on an error when reading a Part content.
```

Comment:
Ideally we'd have some sort of doc-test example that shows how to use this, but I know that's a significant amount of work.  Could we at least point the user at the unit tests for examples?  Otherwise it's really not clear how to use this.

Reply:
I think that would be helpful.  parser.go is pretty short, so I'd be OK with putting it there.

If it's likely we add more of these in the future, a new package might be nice to give them their own Go doc section, and make it easier to copy/paste into your ones own code for modification.